lib/index.js

```diff
@@ -0,0 +1,96 @@
+const config = require('./config');
+
+var request = require('request-promise-native');
+
+
+module.exports = function(apikey) {
+  this.key = apikey;
+
+  this.samlEntity = function(name, entityId, federationId) {
+    this.name = name;
+    this.entityId = entityId;
+    this.federationId = federationId;
+    this.type = "SAML";
+  };
+
+  this.openAthensEntity = function(name, entityId, organizationId, scope) {
+    this.name = name;
```

Comment:
Perhaps ensuring these datapoints are non-null or empty would make sense. I am not sure of the best way to do this in JS but something like:
`
if (!entityId) {
throw("An entity ID is required to create an Open Athens Entity")
}
`